app-connector-client/server/config.js

```diff
@@ -56,10 +57,26 @@ function configValidation(configJson) {
             }
         }
     }
+    if (apis) {
+        for (var i = 1; i <= apis.length; i++) {
+            {
+                var api = apis[i - 1];
+
+                if (!api.authtype) {
+                    error_message += "\napi " + (api.name ? api.name : "number " + i) + ": missing attribute 'authtype', authtype is mandatory";
```

Comment:
We should have a default instead, maybe "none"?

Reply:
no check for auth element exists.. if doesn't exist the default is none